whitelabelco/sofo-foods
# Cp/feat/update history images

### Description 💻

Resolves: #180 

### Implementation 💡

Updates history images

[Comps](https://www.figma.com/design/04vc8B8NFKlryNS0e1h57T/SoFo-Foods?node-id=153-1437&node-type=frame&t=rpWkjn7sRlrBYqzF-0)

#### Type of change


### How to test 🔍

- `npm i`
- `npm run dev`
- navigate to `http://localhost:3000/`
- scroll down and click "Our History"
- confirm images match new [comps](https://www.figma.com/design/04vc8B8NFKlryNS0e1h57T/SoFo-Foods?node-id=153-1437&node-type=frame&t=rpWkjn7sRlrBYqzF-0)

### Screenshots/Demos 📷

### Checklist ✅
